src/plugins/dam/actions/displayBatchEdit.js

```diff
@@ -0,0 +1,6 @@
+import { actionTypes } from '@triniti/cms/plugins/dam/constants';
+
+export default (display) => ({
+  type: actionTypes.DISPLAY_BATCH_EDIT_REQUESTED,
```

Comment:
@rsumilang why is this whole thing in redux? couldnt you handle this with component state in the button?